### PR TITLE
66-chore-use-repo-secrets-for-urls-in-pipeline

### PR DESCRIPTION
**Pull request by JesseMattson**
Replaced URL environment variables for newman in ci-cd.yml ; make test-backend-pipeline is failing

### CODE REVIEW

**Comment by RandallKeur** [comment]
1. Could we also add some documentation around how to add github secrets for future reference so when we need to add more URLs or secrets, we have the documentation readily available? (highly encouraged)
2. Maybe given the errors we were seeing locally, maybe some documentation around Github actions may help too? (Non-blocking)

**Review by RandallKeur — CHANGES_REQUESTED** [review]
I can't highlight it for some reason but downgrade to node version 20.
See tested pipeline [here](https://github.com/JesseMattson/PurrgentCare/actions/runs/10463429520)

File: makefile [hunk 1]
```diff
@@ -31,7 +31,8 @@ start-backend:
 	make build-backend
 	mvn spring-boot:run
 
-test-backend-pipeline: compile-backend test-backend test-postman container-backend
+#TODO use docker to run application so we can run postman without needing two terminals
```
Comment (RandallKeur) [inline makefile:34]: Love this idea, sounds like another feature we can work on next 😄

File: .github/workflows/ci-cd.yml [hunk 2]
```diff
@@ -60,18 +60,18 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: '22'
+          node-version: '20'
```
Comment (JesseMattson) [inline .github/workflows/ci-cd.yml:63]: @RandallKeur let me know if there was anywhere else this needed to be changed.

**Review by RandallKeur — APPROVED** [review]

**Pull request merged**